.github/workflows/ci.yml

```diff
@@ -125,18 +125,19 @@ jobs:
 
       # set arm arch
       - name: Install qemu/docker
-        run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+        run: docker run --privileged --rm tonistiigi/binfmt --install all
```

Comment:
Any reason for all? The image architecture is arm-v7 https://github.com/awslabs/aws-crt-builder/blob/42d15ba4ee3fbe0e812e25094a8cf59eaef0cf1e/.github/docker-images/raspbian-bullseye/Dockerfile#L1.

Reply:
ah, i was being lazy :) 
let me set it to armv7